core/src/styles/fiori-fundamentals-overrides.scss

```
@@ -152,17 +146,10 @@ span {
   }
 }
 
-.fd-action-bar {
-  &__actions {
-    flex-shrink: 0; //another bug when search input is expanded
-    .fd-popover {
-      top: 0; /*  There is a bug from Fiori Fundamentals.
-                  When action bar includes popover with `fd-button--primary` class,
-                  then it's not aligned with other buttons in row anymore,
-                  because it has `top: - fd-space(1)` by default.
-                  Until it's fixed in Fundamentals, better to keep these styles to overwrite it.
-              */
+.y-fd-menu {
+  .fd-menu__item {
+    input[type='checkbox'] {
+      margin-right: 3px; //fix for too small gap between checkbox and its label in popover menu (Filter list)
```

A tiny improvement :)

 ```suggestion
      margin-right: 5px; //fix for too small gap between checkbox and its label in popover menu (Filter list)
```

---

if 8px is fine you could use `@extend .fd-has-margin-right-tiny;` as well. We are already doing that in fiori overrides in lambda repo